### PR TITLE
Translate 3.9 migration guide in Japanese to 3.next

### PR DESCRIPTION
**Pull request by hgsgtk**
This PR is exactly same with https://github.com/cakephp/docs/pull/6225 .
I send 3.9 migration guide translation to 4.x, so resend PR to 3.next.

Later, I am going to send PR to 4.x to delete 3-x-migration-guide.

### CODE REVIEW

File: ja/appendices/3-9-migration-guide.rst [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+* ``Driver::newTableSchema()`` が追加されました。このフックメソッドを使用すると、
+  スキーマメタデータに使用するクラスをカスタマイズすることができます。
+
+Datasource
+----------
+
+* ``Cake\Datasource\SimplePaginator`` が追加されました。このクラスにより、
+  非常に大きな結果のページ分けがより効率的になります。潜在的に高価な ``count()``
+  クエリの実行をスキップします。ページネーションコントロールで「次へ」と「前へ」の
```
Comment (okinaka) [inline ja/appendices/3-9-migration-guide.rst]:
```suggestion
  クエリーの実行をスキップします。ページネーションコントロールで「次へ」と「前へ」の
```

File: ja/appendices/3-9-migration-guide.rst [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+* ``Table::deleteMany()`` および ``Table::deleteManyOrFail()`` メソッドが、
+  コールバックを含む多くのエンティティーを一度に削除するために追加されました。
+  エンティティーはトランザクション内で削除されます。
+* ``TableLocator::clear()`` は内部の ``options`` 配列をリセットするようになりました。
+
+TestSuite
+---------
+
+* ``TestCase::getFixtures()`` が追加されました。このメソッドを使用すると、
+  必要に応じてアプリケーション固有のロジックを使用してフィクスチャリストを生成できます。
```
Comment (okinaka) [inline ja/appendices/3-9-migration-guide.rst]:
```suggestion
  必要に応じてアプリケーション固有のロジックを使用してフィクスチャーリストを生成できます。
```

**Comment by hgsgtk** [comment]
@okinaka thks!

**Comment by okinaka** [comment]
@hgsgtk Thank you!

**Pull request merged**